src/main/java/fr/techad/edc/httpd/search/ContentSearcher.java

```diff
@@ -87,4 +107,20 @@ private void createSearcher() throws IOException {
       indexSearcher = new IndexSearcher(reader);
     }
   }
+
+  private String getdefaultlang() throws IOException {
```

Comment:
Not Java compliant : getDefaultLanguage
And the SearchService search only. This should be read in the handler